lib/groupWith/index.js

```diff
@@ -1,7 +1,9 @@
 import curry2 from '../_internal/_curry2'
 
 export default curry2(function groupWith(func, list) {
-  if (list.length === 0 || !Array.isArray(list)) return []
+  if (list.length === 0 || !Array.isArray(list)) {
```

Comment:
Here and below - just moving of letters. No effect, all the same because all code will be uglified.